scripts/esbuild/sys-node.ts

```diff
@@ -96,6 +96,15 @@ async function sysNodeExternalBundles(opts: BuildOptions) {
     bundleExternal(opts, opts.output.devServerDir, cachedDir, 'ws.js'),
   ]);
 
+  /**
+   * Some of globs dependencies are using imports with a `node:` prefix which
+   * is not supported by Jest v26. This is a workaround to remove the `node:`.
+   * TODO(STENCIL-1323): remove once we deprecated Jest v26 support
+   */
+  const globOutputPath = join(opts.output.sysNodeDir, 'glob.js');
+  const glob = fs.readFileSync(globOutputPath, 'utf8');
+  fs.writeFileSync(globOutputPath, glob.replace(/require\("node:/g, 'require("'));
```

Comment:
Is it better to just revert glob and block that upgrade until we drop Jest 26 support (in v5)?

Reply:
Unfortunately that wouldn't work as `glob` itself is not the culprit here, it's `glob` dependencies, e.g. `path-scurry`, that bring in these node imports. If we would use pnpm we could use the [`overwrite`](https://pnpm.io/package_json#pnpmoverrides) or in Yarn we could use [`resolutions`](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) but there is nothing in NPM to do this. Hence tweaking the build artifact seemed like the easiest approach.